### PR TITLE
implemented inline jwts for operator, this simplifies generate config.

### PR DESCRIPTION
**Pull request by aricart**
Also implemented a hidden mode where the config references jwt files - requires server support.

usage on nsc is always available from nsc --help or by drilling:
```bash
nsc add operator -n O
nsc edit operator --service-url nats://localhost:4222
nsc add account -n A
nsc add user -n aa
nsc add account -n B
nsc add user -n bb

nsc generate config --mem-resolver --config-file /tmp/server.conf
```
On another terminal:
```bash
nats-server -c /tmp/server.conf
```

On another terminal:
```bash
nsc sub foo
```

On yet another:
```bash
nsc pub foo
```

### CODE REVIEW

File: cmd/memresolverconfigbuilder_test.go [hunk 1]
```diff
@@ -130,20 +140,33 @@ func Test_MemResolverErrorBadClaim(t *testing.T) {
...
-	d, err := builder.Generate("./test/relative.jwt")
+func Test_MemResolverDir(t *testing.T) {
+	ts := NewTestStore(t, "O")
+	defer ts.Done(t)
+	ts.AddAccount(t, "A")
+	ts.AddAccount(t, "B")
+
+	out := filepath.Join(ts.Dir, "conf")
+	_, _, err := ExecuteCmd(createServerConfigCmd(), "--mem-resolver",
+		"--dir", out)
```
Comment (sasbury) [inline cmd/memresolverconfigbuilder_test.go]: I wonder if there is a reasonable way to test the -- option, which i assume sends to std out.

Comment (aricart) [reply cmd/memresolverconfigbuilder_test.go]: added one

**Comment by aricart** [comment]
Added a script that you can use to test on the PR description.

**Comment by wallyqs** [comment]
from the example:
```
nsc edit operator --service-url nats://localhost:4222
```

I get: 
```
Error: unknown flag: --service-url
```

File: cmd/common_test.go [hunk 2]
```diff
@@ -25,6 +25,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/spf13/cobra"
+
 	"github.com/mitchellh/go-homedir"
```
Comment (wallyqs) [inline cmd/common_test.go]: does the `nsc` command always rely on the home directory?

**Comment by aricart** [comment]
try installing to make sure you are not getting one of the releases
```
➜ nsc edit operator --service-url nats://localhost:4242
Success! - edited operator
-----BEGIN NATS OPERATOR JWT-----
eyJ0eXAiOiJqd3QiLCJhbGciOiJlZDI1NTE5In0.eyJqdGkiOiI3WlhNNUVJS0xLRUhKVDVTTjZETzdUU0NMNkE3SkM0NDNaVzc2SVFTSFZURk9NVVFUNE5RIiwiaWF0IjoxNTYxNDkyODQxLCJpc3MiOiJPQ1VSSURSU1FWU0FXUEEzR1UyRkwyU1M3NUw3VEVVQVhETERBTUZaUk1PM1dPRFpJU1JZQ05ZSCIsIm5hbWUiOiJPT08iLCJzdWIiOiJPQ1VSSURSU1FWU0FXUEEzR1UyRkwyU1M3NUw3VEVVQVhETERBTUZaUk1PM1dPRFpJU1JZQ05ZSCIsInR5cGUiOiJvcGVyYXRvciIsIm5hdHMiOnsib3BlcmF0b3Jfc2VydmljZV91cmxzIjpbIm5hdHM6Ly9sb2NhbGhvc3Q6NDIyMiIsIm5hdHM6Ly9sb2NhbGhvc3Q6NDI0MiJdfX0.nI-VAjBx2LGJzKxBQ9Otp-7Ut7tFhiaH-tOpilFf4qG210cjjDcfVxt0gX5D-9Rw5P6GM5lVWj32We1V3wBqAw
------END NATS OPERATOR JWT------
```

File: cmd/common_test.go (hunk 2 above)
Comment (wallyqs) [reply cmd/common_test.go]: ah ok can be set via `$NSC_HOME`

Comment (aricart) [reply cmd/common_test.go]: yes for path resolution - in various prompts you can enter ~/somepath if you are managing your keys by hand. Or for output files etc.

**Pull request merged**